src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -42,6 +46,38 @@ public class DurableOrchestrationClient : DurableOrchestrationClientBase
             this.attribute = attribute;
         }
 
+        /// <summary>
+        /// Constructor for <see cref="DurableOrchestrationClient"/>
+        /// </summary>
+        /// <param name="connectionName">Azure Storage connection name.</param>
+        /// <param name="taskHubName">Name of task hub.</param>
+        public DurableOrchestrationClient(string connectionName, string taskHubName)
+        {
+            this.config = new DurableTaskExtension();
+            JobHostConfiguration jobHostConfiguration =
+                new JobHostConfiguration(connectionName)
+                {
+                    LoggerFactory = new LoggerFactory(),
+                };
+            ExtensionConfigContext extensionConfigContext = new ExtensionConfigContext
+            {
+                Config = new JobHostConfiguration(connectionName),
+            };
+            ((IExtensionConfigProvider)this.config).Initialize(extensionConfigContext);
+            this.attribute = new OrchestrationClientAttribute
+            {
+                TaskHub = taskHubName ?? this.config.HubName,
+                ConnectionName = connectionName ?? this.config.AzureStorageConnectionStringName ?? ConnectionStringNames.Storage,
+            };
+            AzureStorageOrchestrationServiceSettings settings = this.config.GetOrchestrationServiceSettings(this.attribute);
+            var serviceClient = new AzureStorageOrchestrationService(settings);
+            this.client = new TaskHubClient(serviceClient);
+
+            ILoggerFactory loggerFactory = new LoggerFactory();
```

Comment:
You created another `LoggerFactory` previous to this. Why not reuse that one?

Reply:
Thank you for catching this! I will improve it.